src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -189,13 +189,19 @@ struct NormalOneForm : db::ComputeItemTag {
   using argument_tags = tmpl::list<DxRadius<Frame>, Rhat<Frame>>;
 };
 
-/// `Tangents(j,i)` is \f$\partial x_{\rm surf}^i/\partial q^j\f$,
-/// where \f$x_{\rm surf}^i\f$ are the Cartesian coordinates of the surface
-/// (i.e. `CartesianCoords`)
-/// and are considered functions of \f$(\theta,\phi)\f$;
-/// \f$\partial/\partial q^0\f$ means \f$\partial/\partial\theta\f$;
-/// and \f$\partial/\partial q^1\f$ means
-/// \f$\csc\theta\,\,\partial/\partial\phi\f$.
+/// `Tangents(i,j)` is \f$\partial x_{\rm surf}^i/\partial q^j\f$,
+/// where \f$x_{\rm surf}^i\f$ are the Cartesian coordinates of the
+/// surface (i.e. `CartesianCoords`) and are considered functions of
+/// \f$(\theta,\phi)\f$. \f$\partial/\partial q^0\f$ means
```

Comment:
Could you end the "brief" part of the documentation at this period? This means just adding a `///` line after the period. The brief is extreme long in this case.